src/main/resources/kn.json

```diff
@@ -1,30 +1,30 @@
 {
   "tools": {
     "kn": {
-      "version": "1.5.0",
+      "version": "20220823-local-6d86bf75",
```

Comment:
I think it would be better to have 1.7.0 as it is more comprehensive and will probably uses as the folder name